phabricatoremails/model/phabricator.py

```diff
@@ -0,0 +1,150 @@
+from dataclasses import dataclass
+from datetime import datetime
+from enum import Enum
+from typing import Optional, List, Union
+
+from phabricatoremails.model.common import Recipient, Reviewer
+
+
+@dataclass
+class Bug:
+    id: int
+    name: str
+    link: str
+
+
+@dataclass
+class Revision:
+    id: str
+    name: str
+    link: str
+    bug: Optional[Bug]
+
+
+@dataclass
+class ReplyContext:
+    other_author: str
+    other_date_utc: datetime
+    other_comment: str
+
+
+class DiffLineType(Enum):
+    ADDED = "added"
+    REMOVED = "removed"
+    NO_CHANGE = "no-change"
+
+
+@dataclass
+class DiffLine:
+    number: int
+    type: DiffLineType
+    raw_content: str
+
+
+@dataclass
+class CodeContext:
+    diff: List[DiffLine]
+
+
+@dataclass
+class InlineComment:
+    file_context: str
+    link: str
+    text: str
+    context: Union[ReplyContext, CodeContext]
+
+
+class AffectedFileChange(Enum):
```

Comment:
Same comment as `DiffLineType`, and all other enums defined in the PR.

Reply:
Resolving this conversation and joining it with [this one](https://github.com/mozilla-conduit/phabricator-emails/pull/1#discussion_r403292804)